src/Discovery/aggMDSUtils.jsx

```diff
@@ -72,13 +72,14 @@ const loadStudiesFromAggMDSRequests = async (offset, limit) => {
       x.study_id = studyId;
       x.commons = commonsName;
       x.frontend_uid = `${commonsName}_${index}`;
-      x.short_name = x.name;
       x._unique_id = `${commonsName}_${x._unique_id}_${index}`;
       x.commons_url = commonsInfo.commons_url;
       x.tags.push(Object({ category: 'Commons', name: commonsName }));
+      x.name = x.short_name; // TODO: this will need to be refactored
```

Comment:
I think the assumtion in the old logic is `short_name` can be optional, and make it the same as `name` if omitted
Looks like that is no longer true?

Reply:
There is some issue with short_name as all of the metadata returns this field. This is actually a bigger issue in that we:
* allow Ecosystem Discovery to have its fields completely configurable
or
* Define a spec for minimal set of expected fields from the metadata services

Reply:
I actually would vote for the 2nd option, as we are already having a number of "predefined meaningful" fields, and will be likely to have more, if we can have a defined "minimal set of meaningful fields", it can both simplify portal config and data ingestion